ooiui/templates/science/plotting.html

```diff
@@ -362,6 +363,7 @@
         options.xvar = 'time';
       }
       
+      console.log(options.yvar)
```

Comment:
Also you want me to remove all the console.logs in those files or just the ones I added?